onnxscript/ir/__init__.py

```diff
@@ -1,6 +1,4 @@
 """In-memory intermediate representation for ONNX graphs."""
-from __future__ import annotations
```

Comment:
Keep?

Reply:
was introduced by merging w/ `__init__.py` from legacy ir. If needed we can add in follow ups.